ui/cypress/component/DirectiveEdit.e2e.cy.js

```diff
@@ -41,10 +41,12 @@ describe('FullScreenTemplater: directive', { browser: ['chrome', 'chromium', 'fi
 
   beforeEach(() => {
 
+    // TODO check with remote/auth:
```

Comment:
I did not update these two component tests to work with auth / remove intances (yet?). The team also realized that we might remove basic auth from remote instances and replace with keycloak, at which point the TODO will be to login on tests before proceeding.